src/exchange/tradingPair/TradingPairView.scss

```diff
@@ -104,37 +109,70 @@
 
 
 
-// ------------ Pair view -----------------
+// ------------ Active pair view -----------------
 
-.pairView {
+.activePairView {
     display: flex;
     justify-content: space-between;
     align-items: center;
     padding: 12px 26px 12px 12px;
     width: 100%;
 }
 
-.pairViewCurrency {
+.activePairViewIcon {
+    height: 26px;
+    width: 26px;
+    margin-right: 20px;
+    display: flex;
+    align-items: center;
+}
+
+.activePairViewCurrency {
     flex-basis: 50px;
     flex-grow: 1;
 }
 
-.pairViewIcon {
-  height: 26px;
-  width: 26px;
-  margin-right: 20px;
-  display: flex;
-  align-items: center;
+// ------------ Pair view -----------------
+
+.pairView {
+    display: table-row;
```

Comment:
What's the point of having the elements as `table-row`? If it's regarding some alignment use `flex`

Reply:
Flex does not work with rows and columns!
Alternatively I could use grid, but it would cause problems with table rows as links.

Reply:
My point is to not use `table-row` at all

Reply:
How would you align without `table-row`?

Reply:
With `inline-flex`. You can apply flex